Containers/talk/Dockerfile

```diff
@@ -1,18 +1,17 @@
 FROM nats:2.9.18-scratch as nats
 FROM strukturag/nextcloud-spreed-signaling:1.1.2 as signaling
-FROM coturn/coturn:4.6.2-r3-alpine
-USER root
-# Pin alpine version manually as long as https://github.com/coturn/coturn/issues/1226 is not done
-ENV ALPINE_VERSION=3.18
+FROM ghcr.io/processone/eturnal:x.x.x-otpxx AS eturnal
+FROM alpine:3.18.2
 
```

Comment:
```suggestion
ENV STUN_SERVICE="stun.nextcloud.com 443"


```

Reply:
And revert this in the Dockerfile as well then:
https://github.com/processone/eturnal/issues/42#issuecomment-1605648407

Only related to the `eturnalctl` part though.

Reply:
> And revert this in the Dockerfile as well then: [processone/eturnal#42 (comment)](https://github.com/processone/eturnal/issues/42#issuecomment-1605648407)
> 
> Only related to the `eturnalctl` part though.

what do you exasctly mean? are ln -s not required?

Reply:
Apparently with https://github.com/nextcloud/all-in-one/pull/2855#discussion_r1241233958 we actually dont need the autodiscovery anymore.

Reply:
`eturnalctl` in `/opt/eturnal/bin` does not have the STUN query included by default, hence sym-linking like now will not lead to the desired result of performing the STUN query.

The `eturnalctl` in our Docker image contains the [STUN query](https://github.com/sando38/eturnal/blob/ca13a4fb4ead2768302f6c37e97144dcbafa37ec/docker-k8s/image/Dockerfile#L117-L126), hence you should COPY it from it instead of sym-linking in your RUN command. 

So remove from RUN:
```
ln -s /opt/eturnal/bin/eturnalctl /usr/local/bin/eturnalctl
```
and add:
```
COPY --from=eturnal /usr/local/bin/eturnalctl /usr/local/bin/eturnalctl
```